docs/element-templates/README.md

```diff
@@ -19,7 +19,7 @@ Element templates allow you to create pre-defined configurations for BPMN elemen
 
 Element templates are defined as [JSON files](#defining-templates). There are two ways to integrate them into the Camunda Modeler:
 
-1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo.
+1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Locally the sycned templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](https://github.com/camunda/camunda-modeler/tree/master/docs/search-paths#user-data-directory). Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well).
```

Comment:
> sycned

Typo.

Reply:
ty :+1:

Reply:
BTW I'm not sure if this is detailed enough to actually remove the templates manually.